R/api.R

```diff
@@ -46,15 +46,23 @@ metadata <- function() {
   get_option <- function(id, label) {
     list(id = scalar(id), label = scalar(label))
   }
+
+  param_ids <- lapply(response$parameters, function(param) {
+    param$id
+  })
+
   # Set available countries from daedalus package
   # JIDEA-62: use the right version of daedalus/model
   # we will get ISO ids from daedalus when available
-  country_options <- lapply(daedalus::country_names, function(country) {
+  country_names <- daedalus::country_names
+
+  country_options <- lapply(country_names, function(country) {
     country_string <- as.character(country)
```

Comment:
Not in this PR but `country` is a character and this line can be cut.

Reply:
Hm, this might be a hangover from previous data frame implementation, maybe an attempt to get round single vector boxing? Not sure, will check if I can get rid of it.

Reply:
Again, whatever problem I thought I was solving here seems to have disappeared, and removing `as.character` still works. Thanks!